frontend/src/app/organization/organization.service.ts

```diff
@@ -49,6 +51,24 @@ export class OrganizationService {
       );
   }
 
+  createOrganization(newOrganization: Organization) {
+    this.http.post(this.url, newOrganization)
+      .subscribe(
+        (data: HttpResponse<Organization>) => {
+          this.createOrganizationEvent.next({data, status:'success'});
```

Comment:
You are using the same event for createOrganization as well as for editOrganization, but there is no difference in data structure you send to the next function. 
I would send at least `{data, type: 'create'}` for this method and `{data, type: 'edit'}` for editOrganization. Also `{data, type: 'error}` for error handlers. I would define it as an interface in organization.interfaces. Don't forget to adhere to this change in a subscription function you execute in the component.